x-pack/plugins/security_solution/public/timelines/components/timeline/query_tab_content/index.tsx

```diff
@@ -404,6 +420,22 @@ const mapDispatchToProps = (dispatch: Dispatch, { timelineId }: OwnProps) => ({
       })
     );
   },
+  onEventClosed: ({ indexName, eventId }: { indexName: string; eventId: string }) => {
+    dispatch(
+      timelineActions.toggleExpandedEvent({
+        timelineId,
+        event: {},
+      })
+    );
+
+    if (timelineId === TimelineId.active) {
+      activeTimeline.toggleExpandedEvent({
```

Comment:
I'm not sure if accessing `activeTimeline` in mapDispatchToProps is a good idea, because whenever this value change the whole component will rerender